apps/src/AzureTextToSpeech.js

```diff
@@ -0,0 +1,252 @@
+import {hashString, findProfanity} from '@cdo/apps/utils';
+
+// XMLHttpRequest readyState 4 means the request is done.
+// https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/readyState
+const READY_STATE_DONE = 4;
+
+export class SoundResponse {
```

Comment:
Could you add a bit of documentation explaining what a `SoundResponse` is, and how it's used?